packages/backend/server/src/models/user.ts

```diff
@@ -14,7 +14,7 @@ import {
 } from '../base';
 import type { Payload } from '../base/event/def';
 import { Permission } from '../core/permission';
-import { Quota_FreePlanV1_1 } from '../core/quota';
+import { Quota_FreePlanV1_1 } from '../core/quota/schema';
```

Comment:
@forehalo fix the dependency cycle

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/hTwOityLamd4hitrae7M/045be7eb-f826-4ae4-b497-4bc2aab9b338.png)

Reply:
will migrate service/controller/resolver usage after all models get ready